src/providers/twitch/api/Helix.cpp

```diff
@@ -761,6 +761,37 @@ void Helix::getCheermotes(
         .execute();
 }
 
+void Helix::getEmoteSetData(QString emoteSetId,
+                            ResultCallback<HelixEmoteSetData> successCallback,
+                            HelixFailureCallback failureCallback)
+{
+    QUrlQuery urlQuery;
+
+    urlQuery.addQueryItem("emote_set_id", emoteSetId);
+
+    this->makeRequest("chat/emotes/set", urlQuery)
+        .onSuccess([successCallback, failureCallback,
+                    emoteSetId](auto result) -> Outcome {
+            QJsonObject root = result.parseJson();
+            auto data = root.value("data");
+
+            if (!data.isArray())
+            {
+                failureCallback();
+                return Failure;
+            }
+
+            HelixEmoteSetData emoteSetData(data.toArray()[0].toObject());
+
+            successCallback(emoteSetData);
+            return Success;
+        })
+        .onError([](NetworkResult result) {
+            qDebug() << "Epix helix fail xd" << result.status();
+        })
```

Comment:
I believe this qDebug to be unspecific and unhelpful for debugging. Please change or remove it :)

Reply:
Same deal as with all other debugs; I wrote this code at 4am ok? ;v thanks for pointing out though